gef.py

```diff
@@ -900,26 +901,41 @@ def is_exe(fpath):
     raise FileNotFoundError("Missing file `{:s}`".format(program))
 
 
-def hexdump(source, length=0x10, separator=".", show_raw=False, base=0x00):
+def hexdump(source, length=0x10, separator=".", show_raw=False, base=0, fmt='byte'):
     """Return the hexdump of `src` argument.
-    @param source *MUST* be of type bytes or bytearray
+    @param mem is the data to dump, bytes or bytearray only
     @param length is the length of items per line
     @param separator is the default character to use if one byte is not printable
     @param show_raw if True, do not add the line nor the text translation
     @param base is the start address of the block being hexdump
-    @param func is the function to use to parse bytes (int for Py3, chr for Py2)
     @return a string with the hexdump """
+    formats = {
+        "qword": ("Q", 8),
+        "dword": ("I", 4),
+        "word": ("H", 2),
+        "byte": ("B", 1),
+    }
+    type = formats[fmt][0]
```

Comment:
Can you rename this variable, since `type` is already a Python builtin (I know that if it's not a callable the VM will differenciate, I just prefer avoiding naming collision)?